gradle-mvn-push.gradle

```diff
@@ -32,11 +32,21 @@ def getSnapshotRepositoryUrl() {
 }
 
 def getRepositoryUsername() {
-  return hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : ""
```

Comment:
Ah, didn't know about that, thanks!

So basically, this pulll request can be rejected and @EricKuck only has to set:

````
travis encrypt -r bluelinelabs/Conductor "ORG_GRADLE_PROJECT_NEXUS_USERNAME=InsertYourUsername" --add
travis encrypt -r bluelinelabs/Conductor "ORG_GRADLE_PROJECT_NEXUS_PASSWORD=InsertYourPassword" --add
```

right?